### ISSUE TITLE
USER STORY: Model optimization V6

### ISSUE THREAD

**Issue by Huddy2022** [Huddy2022/milestone-project-mildew-detection-in-cherry-leaves#25]
As a **data scientist** I want to **train the CNN model using the pre-processed dataset with re-sizes images 100 x 100 and optimize the hyperparameters min value 128, max value 256, step 32, filters 32, 64 & 128, dropout 0.3, with values 1e-3, 1e-4, 1e-5, on a Relu activation, loss of binary_crossentropy, add regularizers L1, L2 & none, but also add Batchnormalization on all Conv2D to achieve the best performance,** so that **the model can accurately classify cherry leaves as healthy or containing powdery mildew.**
